eng/tools/rush-runner.js

```diff
@@ -186,6 +187,15 @@ if (serviceDirs.length === 0) {
       }
       break;
 
+    case "build":
+      if (actionComponents[1] === "samples") {
+        for (const packageDir of packageDirs) {
+          spawnNode(packageDir, "../../../common/scripts/install-run-rushx.js", action);
+        }
+        break;
+      }
+    // Intentional fall-through
```

Comment:
I am little skeptical of this approach. This will get easily messed up if someone adds a new case or changes the order in the future. Isn't this build action not working with below default case

Reply:
In effect what I intended for this to do is to special-case `build:samples`. So all other `build` commands will fall through. Indeed it will break if someone adds another case after `case "build"` and before `default`, but I will need some guidance on how to modify this file or how to avoid using rush-runner altogether otherwise.